components/Deck/ContentModulesPanel/DataSourcePanel/EditDataSource.js

```diff
@@ -22,7 +22,7 @@ class EditDataSource extends React.Component {
             if (!url.startsWith('http://') && !url.startsWith('https://')) {
                 url = 'http://' + url;
             }
-            let expression = /https?:\/\/(www\.)?[-a-z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-z0-9@:%_\+.~#?&//=]*)?/i;
+            let expression = /https?:\/\/(www\.)?[-a-z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b(\/[-a-z0-9@:%_\+.~#?&//=]*)?/i;
```

Comment:
why include `//` (twice) inside the last brackets `[]` part ?
and, I still don't get why we need the `\b` (word boundary) there

Reply:
@kprist yes, after checking with http://regex101.com  the '//' was a mistake, but the \b, for some reason I don't understand completely well, is the one that doesn't allow to enter urls like 'google.com123' or 'google.commmmmmmmmmmm'... even when the {2,6} is specified there.